Update 0069-VCBS-validators_chosen_by_stake.md (#1434)
vegaprotocol/specs: protocol/0069-VCBS-validators_chosen_by_stake.md

  * Announce 2 new nodes but self-delegate only to one of them. 
  * Verify that, after 1000 blocks and on the following epoch, only the validator to which we self-delegated got promoted and we now have 4 Tendermint validators and 1 pending validator. 

> 11. Change ownstake requirement 

Reviewer: ```suggestion
11. Change ownstake requirement (<a name="0069-VCBS-050" href="#0069-VCBS-050">0069-VCBS-050</a>)
```

Verify that one validator is replaced the following epoch, one in the epoch after

13.  Number of slots decreased (<a name="0069-VCBS-052" href="#0069-VCBS-052">0069-VCBS-052</a>):
  * Setup a network with 7 Tendermint validators, self-delegate to them (set the parameter `network.validators.tendermint.number` to 5, set the `network.validators.ersatz.multipleOfTendermintValidators` parameter to 0 so there are no ersatz validators allowed).
  * Decrease the number of tendermint validators to 5.

>   * Verify that in each of the following two epochs, the validator with the lowest score is demoted to Ersatzvalidator and an Ersatzvalidator is demoted to p[endiong

Reviewer: ```suggestion
  * Verify that in each of the following two epochs, the validator with the lowest score is demoted to Ersatzvalidator and an Ersatzvalidator is demoted to pending
```

14. Number of Ersatzvalidators increased (<a name="0069-VCBS-053" href="#0069-VCBS-053">0069-VCBS-053</a>):
  * Setup a network with 6 Tendermint validators, 3 ErsatzValidators (network.validators.ersatz.multipleOfTendermintValidators = 0.5), and 4 pending validators
  * Change the parameter network.validators.ersatz.multipleOfTendermintValidators to 0.9
  * Verify that in the following two epochs, in each epoch the ErsatzValidator with the highest score is promoted to Validator
  * Verify that the third Ersatzvalidator is not promoted in the third epoch

15. Number of Ersatzvalidators decreased (<a name="0069-VCBS-054" href="#0069-VCBS-054">0069-VCBS-054</a>):
  * Setup a network with 6 Tendermint validators, 3 ErsatzValidators (network.validators.ersatz.multipleOfTendermintValidators = 0.5)
  * Change the parameter network.validators.ersatz.multipleOfTendermintValidators to 0.1
  * Verify that in the following to epochs, in each epoch the ErsatzValidator with the lowest score is demoted to pending 
  * Verify that the third Ersatzvalidator is not promoted in the third epoch

> 16. Number of Ersatzvalidators Eratic (<a name="0069-VCBS-055" href="#0069-VCBS-055">0069-VCBS-055</a>):

Reviewer: ```suggestion
16. Number of Ersatzvalidators Erratic (<a name="0069-VCBS-055" href="#0069-VCBS-055">0069-VCBS-055</a>):
```

  * Setup a network with 6 Tendermint validators, 3 ErsatzValidators (network.validators.ersatz.multipleOfTendermintValidators = 0.5), and 4 pending validators
  * Change the parameter network.validators.ersatz.multipleOfTendermintValidators to 0.9
  * In the next epoch, change network.validators.ersatz.multipleOfTendermintValidators to 0.1
  * Two epochs lated, change network.validators.ersatz.multipleOfTendermintValidators to 0.5
  * Verify that in the following four epochs, first a pending validator is promoted, then two pending validators are demoted, then one is promoted agian (eith the highest/losest scores respectively)
  * Verify that in the fifth epoch, no demotions/promotions happen and the number of Ertsatzvalidators stays at 3

> 17. Number of ErsatzValidators oddly define (<a name="0069-VCBS-056" href="#0069-VCBS-056">0069-VCBS-056</a>)d

Reviewer: ```suggestion
17. Number of ErsatzValidators oddly defined (<a name="0069-VCBS-056" href="#0069-VCBS-056">0069-VCBS-056</a>)d
```

  * Set the factor to 0.00000000000000000000000000000000000000001
  * Verify that all Validators round it the same way, and that there are no Ersatzvalidators

>   * Set the factor to 3.00000000000000000000000000000000000000001

Reviewer: ```suggestion
  * Set the factor to 3.00000000000000000000000000000000000000001 and run the network with just one tendermint (consensus) validator.
```

>   * Verify that all Validators round it the same way, and that there are three Ersatzvalidators

Reviewer: We need to say how many consensus / tendermint validators there are, don't we? Make it 1 tendermint if we want 3 ersatz?
